x-pack/plugins/alerting/public/pages/maintenance_windows/components/create_maintenance_windows_form.tsx

```diff
@@ -35,16 +43,20 @@ export interface CreateMaintenanceWindowFormProps {
   maintenanceWindowId?: string;
 }
 
-export const useTimeZone = (): string => {
-  const timeZone = useUiSetting<string>('dateFormat:tz');
-  return timeZone === 'Browser' ? moment.tz.guess() : timeZone;
+const useDefaultTimzezone = () => {
+  const kibanaTz: string = useUiSetting('dateFormat:tz');
+  if (!kibanaTz || kibanaTz === 'Browser') {
```

Comment:
it might be nice to show the timezone (without being able to change it) if it's set in the advanced settings.